### PR TITLE
Rename 'devicePlugin' to 'rdmaSharedDevicePlugin'

### PR DESCRIPTION
**Pull request by e0ne**
Since we will allow configure and install SR-IOV Network Device Plugin
we need to have more user-friendly name to not confuse users with a
'devicePlugin' option.

### CODE REVIEW

File: deployment/network-operator/templates/tests/test-dev-plugin.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.deployCR .Values.devicePlugin.deploy  }}
+{{- if and .Values.deployCR .Values.rdmaSharedDevicePlugin.deploy  }}
```
Comment (adrianchiris) [inline deployment/network-operator/templates/tests/test-dev-plugin.yaml:1]: a comment about the name of the file, should we change to `test-rdma-shared-dev-plugin.yaml` ?
im fine either way, but thought its worth mentioning

**Review by adrianchiris — APPROVED** [review]

File: deployment/network-operator/templates/tests/test-dev-plugin.yaml (hunk 1 above)
Comment (e0ne) [reply deployment/network-operator/templates/tests/test-dev-plugin.yaml:1]: Good catch, Adrian! I'll rename it once CI reports on it

**Comment by AbdYsn** [comment]
@e0ne CI is passing with mellanox/kubernetes-ci#93

**Comment by AbdYsn** [comment]
@moshe010 we implemented a new mechanism in jenkins to be able to checkout ci patchs to test api breaking patchs. it worked in this pr using `depends-on: kubernetes-ci#93` in the commit message.

**Review by moshe010 — APPROVED** [review]

**Pull request merged**